### ISSUE TITLE
Training details 

### ISSUE THREAD

**Issue by benedettaliberatori** [ktr-hubrt/MPN#27]
Hi,

First, congratulations for the great paper and thank you for publishing the code.

Unfortunately, I'm not able to reproduce the results in the unsupervised setting described in the paper on the ShanghaiTech dataset. As already described in #20, in the paper you mentioned that you first pre-train using only the frame prediction loss.

So, I did the following steps:
- I downloaded the ShanghaiTech dataset, whose training is composed of videos.
- For each of these videos, I created a folder named as the base name of the file, where I extracted the frames using ffmpeg. For example, for the video `01_001.avi` I created a folder named `01_001` and I extracted the frames using the command `ffmpeg -r 1 -i 01_001.avi -r 1 "01_001/%04d.jpg"`. As a result, the `training` folder is organized in the same way as the `testing` folder.
- Instead of using the `VideoDataLoader`, which is used in your `Train.py`, I used your `DataLoader` class because this data loader iterates through all frames.
- I trained from scratch your model (the `convAE` class) for 10 epochs using only the frame prediction loss. 

With these steps the frame prediction loss does not converge.  Is this pre-training step correct or you did something different? After this pre-training step and before the meta-training, do you train again your model using the code in `Train.py`?

Many thanks in advance.
